var/spack/repos/builtin/packages/exa/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Exa(Package):
+
```

Comment:
I would also get rid of this blank line

Reply:
I make sure to re run the tests and fix any problems in my next commit.